cypress/support/pages/home.pageObject.js

```diff
@@ -4,7 +4,17 @@ class HomePageObject extends PageObject {
   url = '/#/';
 
   get usernameLink() {
-    return cy.getByDataCy('username-link');
+    return cy.getByDataCy('usernameLink');
+  }
+
+  cheackArticlesList() {
```

Comment:
There is a typo in the method name 'cheackArticlesList'. Correct spelling should be 'checkArticlesList'.